public/app/features/dashboard/state/initDashboard.test.ts

```diff
@@ -101,6 +108,19 @@ describeInitScenario('Initializing new dashboard', ctx => {
   ctx.setup(() => {
     ctx.storeState.user.orgId = 12;
     ctx.args.routeInfo = DashboardRouteInfo.New;
+    ctx.storeState.explore.left.originPanelId = 2;
```

Comment:
would like to move this to its own scenario, and not have it be part of the "New dashboard" scenario.

Reply:
Fair enough, had some trouble trying to do this initially but I can give it another go 😅